### PR TITLE
adding footer

### PR DESCRIPTION
**Pull request by OsafAliSayed**
I have added the footer 

before:
image
after:
image

### CODE REVIEW

File: shifter/theme/templates/base.html [hunk 1]
```diff
@@ -2,79 +2,131 @@
...
-        <script>
-            // Timezone settings
-            const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone; // e.g. "America/New_York"
-            document.cookie = "django_timezone=" + timezone;
-        </script>
-    </body>
-</html>
+
+        <div class="text-center ">
+            © 2021 Copyright:
```
Comment (TobySuch) [inline shifter/theme/templates/base.html:119]: Can you change this to 2023 please

**Review by TobySuch — CHANGES_REQUESTED** [review]
Thanks! This is a great start, and I'm sorry the ticket was quite brief. I think the footer can be simplified down though with the following changes:
- Remove the `Links` heading
- On the left side just have the copyright notice
- On the right side just a link to the Github page. I think the text can just be an SVG Invertocat Github logo and the word `Source` next to it. You can grab the SVG from [here](https://github.com/logos) and embed it into the HTML directly.

Thanks!

**Comment by OsafAliSayed** [comment]
did the changes as asked
before: 
image
after:
image

File: shifter/theme/templates/base.html [hunk 2]
```diff
@@ -2,79 +2,120 @@
...
+    {% block title %}<title>Shifter</title>{% endblock %}
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    {% block head %}
+    {% endblock %}
+    {% tailwind_css %}
+</head>
+
+<body class="font-sans leading-normal tracking-normal bg-white">
```
Comment (TobySuch) [inline shifter/theme/templates/base.html:16]: Can you change the body classes to the following:
```
<body class="flex flex-col font-sans leading-normal tracking-normal bg-white h-screen">
```

This is to make the footer appear at the bottom

File: shifter/theme/templates/base.html [hunk 3]
```diff
@@ -2,79 +2,120 @@
...
-        <div class="container mx-auto mt-10">
-            <div class="flex justify-center">
-                {% block content %}
-                <div class="flex items-center justify-center">
-                    <h1 class="text-5xl">Template Page</h1>
-                </div>
-                {% endblock %}
+        </div>
+    </nav>
+    <div class="container mx-auto mt-10">
```
Comment (TobySuch) [inline shifter/theme/templates/base.html:87]: Can you add the `flex-grow` class to this div:
```
<div class="container mx-auto mt-10 flex-grow">
```
to make the footer fix to the bottom of the screen.

**Review by TobySuch — CHANGES_REQUESTED** [review]
Thanks, couple more changes to make the footer appear at the bottom of the page.

**Comment by TobySuch** [comment]
I've moved the base.html file in #177 so you will need to rebase/merge - sorry!

**Review by TobySuch — APPROVED** [review]
Thanks for the contribution, really appreciate it!

**Pull request merged**